app/models/consent/default.rb

```diff
@@ -4,11 +4,21 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# frozen_string_literal: true
+
 class Consent::Default
   def initialize(client:)
     @client = client
   end
 
+  def current_consent_type
+    consent_type = no_release_string
+    consent_type = full_release_string if @client.active_consent_form&.consent_type == full_release_string
+    consent_type = partial_release_string if @client.active_consent_form&.consent_type == partial_release_string
+    consent_type = revoked_consent_string if @client.newest_consent_form&.revoked?
+    consent_type
+  end
```

Comment:
This feels like either duplication of logic that _must_ be somewhere else, or an additional set of logic that is looking back at the source.  If the latter, I think it should only be used in the "cleanup" of existing data, and should also trigger a refresh of the `ClientRoiAuthorization` where it is used.  We want the string on the client to always be in sync with their active `ClientRoiAuthorization`.  (Mostly this is a note to me to watch how this is used, but also, maybe it should be renamed?)